### PR TITLE
Init Spanish translation

### PR DESCRIPTION
**Pull request by raelga**
Init of the Spanish translation of the Kubernetes documentation / Website.

### CODE REVIEW

**Comment by raelga** [comment]
CLA signed.

**Comment by raelga** [comment]
I'm still working on the localization of some sections:

- /partners
- /docs/home
- /docs/concepts
- /docs/concepts/overview/what-is-kubernetes.md

And looking for a Spanish another reviewer on the Slack channel.

File: content/es/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Este Código de Conducta se aplica tanto dentro de los espacios relacionados con el proyecto como en espacios públicos y siempre que un individuo esté representando al proyecto o su comunidad.
+
+Los casos de comportamiento abusivo, acosador o de cualquier otro modo inaceptable podrán ser denunciados poniéndose en contacto con [Comité del Código de Conducta de Kubernetes](https://git.k8s.io/community/committee-code-of-conduct) al email <conduct@kubernetes.io>. Para otros proyectos, comuníquese con un mantenedor de proyectos de CNCF o con nuestra mediadora, Mishi Choudhary <mishi@linux.com>.
```
Comment (alexbrand) [inline content/es/community/static/cncf-code-of-conduct.md]: Should this be: "poniéndose en contacto con el Comité" (missing "el")

File: content/es/docs/home/_index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "Documentación"
+    weight: 20
+    post: >
+      <p>Aprenda a usar Kubernetes con documentación conceptual, tutoriales y manuales de referencia. ¡Incluso puedes <a href="/editdocs/" data-auto-burger-exclude> contribuir con la documentación</a>! </p>
+overview: >
+  Kubernetes es orquestrador de contenedores de código abierto para automatizar la implementación, el escalado y la administración de aplicaciones en contenedores. El proyecto de código abierto está alojado por la Cloud Native Computing Foundation.(<a href="https://www.cncf.io/about">CNCF</a>).
```
Comment (alexbrand) [inline content/es/docs/home/_index.md]: Kubernetes es _un orquestador_

Typo in "orquestrador" -> "orquestador". Also missing "un".

File: content/es/partners/_index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+<section id="users">
+    <main>
+        <h5>Kubernetes trabaja con socios para crear una base de código sólida y vibrante que admita un espectro de plataformas complementarias.</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>Kubernetes Certified Service Providers</b>
+              </h5>
+              <br>Proveedores de servicios con amplia experiencia ayudando a las empresas a adoptar con éxito Kubernetes.
```
Comment (alexbrand) [inline content/es/partners/_index.html]: "adoptar Kubernetes con éxito" suena mejor?

File: content/es/community/static/cncf-code-of-conduct.md (hunk 1 above)
Comment (raelga) [reply content/es/community/static/cncf-code-of-conduct.md]: wow, that's the original one that can only be a exact copy (as stated in the README).

I just opened a PR to update the original: https://github.com/cncf/foundation/pull/34, the we will be able to update the website copy.

Comment (raelga) [reply content/es/community/static/cncf-code-of-conduct.md]: Fixed in https://github.com/cncf/foundation/commit/e2a956a681f089ed9eddb9ba2eadc51f60658596 and updated in 21c69932e27b921cb657a09d7e7ff17e2a8938f0

File: content/es/docs/home/_index.md (hunk 2 above)
Comment (raelga) [reply content/es/docs/home/_index.md]: Fixed in https://github.com/kubernetes/website/pull/13543/commits/4c1b842159089b29b3d6a9f3c179d1393e4dec70

File: content/es/partners/_index.html (hunk 3 above)
Comment (raelga) [reply content/es/partners/_index.html]: Fixed in https://github.com/kubernetes/website/pull/13543/commits/5f332b24a49d9abf288b2f2d661fb55e5485da55

File: content/es/_index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+#### Preparado para trabajar a escala
+
+Diseñado sobre los mismos principios que le permiten a Google ejecutar miles de millones de contenedores cada semana, Kubernetes puede escalar sin aumentar su equipo de operaciones.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="blocks" %}}
+#### Preparado para cualquier complejidad
+
+Ya sea para pruebas en local o para cargas de trabajo para una empresa global, la flexibilidad de Kubernetes crece contigo para entregar sus aplicaciones de manera consistente y fácil sin importar la complejidad.
```
Comment (electrocucaracha) [inline content/es/_index.html]: s/pruebas en local/pruebas locales

File: content/es/community/_index.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+<section id="mainContent">
+    <main>
+        <div class="content">
+            <h3>Garantizando el funcionamiento de Kubernetes para todo el mundo y en cualquier lugar.</h3>
+            <p>Conecte con la comunidad Kubernetes en nuestro canal de <a href="http://slack.k8s.io/">Slack</a>, <a href="https://discuss.kubernetes.io/">foro</a>, o únete al
+                <a href="https://groups.google.com/forum/#!forum/kubernetes-dev">Kubernetes-dev Google group</a>.
+                Cada semana se lleva a cabo una reunión de la comunidad  por videoconferencia para discutir el estado de cosas, revise el documento
+                <a href="https://github.com/kubernetes/community/blob/master/events/community-meeting.md "> Community Meeting </a> para obtener información sobre cómo participar. </p>
+            <p>También puede formar parte de la comunidad en cualquier parte del mundo a través de la
+                <a href="https://www.meetup.com/topics/kubernetes/">Comunidad de Meetups sobre Kubernets </a>o el
```
Comment (electrocucaracha) [inline content/es/community/_index.html]: s/Kubernets/Kubernetes

File: content/es/community/_index.html [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+                Si su interés especial no está cubierto (todavía), también encontrará instrucciones para formar un nuevo SIG.</p>
+            <p>Como miembro de la comunidad Kubernetes, le invitamos a unirse a cualquiera de las reuniones SIG. No necesita registro.</p>
+        </div>
+        <div class="content">
+          <h3>Código de Conducta</h3>
+          <p>La comunidad Kubernetes valora el respeto y la inclusión, y por ello
+              se ha cumplir un <a href="code-of-conduct/">Código de Conducta</a> en
+              cada interacción. Si observa una violación del Código de conducta
+              en un evento o reunión, en Slack, o en otro medio de comunicación,
+              puedes ponerte en contacto con el <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">Comité del Código de Conducta de Kubernetes</a>
```
Comment (electrocucaracha) [inline content/es/community/_index.html]: s/puedes ponerte/puede ponerse

File: content/es/community/static/cncf-code-of-conduct.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+Estamos comprometidos con hacer de la participación de todos los involucrados en este proyecto una experiencia libre de hostigamiento de cualquier índole, sin importar su nivel de experiencia, género, identidad y expresión de género, orientación sexual, discapacidad, aspecto personal, tamaño corporal, raza, grupo étnico, edad, religión ni su nacionalidad.
+
+Algunos ejemplos de conducta inaceptable imputable a los participantes pueden ser:
+
+-	El uso de palabras o imágenes sesgadas de contenido sexual.
+-	Ataques personales
+-	Comentarios provocadores, insultantes o despectivos
+-	Hostigamiento público o privado
+-	Publicar información ajena privada, como ser domicilios particulares o electrónicos, sin contar con permiso explícito para hacerlo
```
Comment (electrocucaracha) [inline content/es/community/static/cncf-code-of-conduct.md]: s/como ser domicilios particulares o electrónicos/como ser domicilios particulares o cuentas electrónicas

File: content/es/docs/sitemap.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+#filters select{
+  font-size: 14px;
+  border: 1px #000 solid;
+}
+#filters {
+  padding-top: 20px;
+}
+</style>
+
+Click tags or use the drop downs to filter. Click table headers to sort or reverse sort.
```
Comment (electrocucaracha) [inline content/es/docs/sitemap.md]: Me parece que este parrafo debe ser traducido

File: README-es.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+make docker-image
+```
+
+Una vez tenga la imagen construida, puede levantar el sitio web ejecutando:
+
+```bash
+make docker-serve
+```
+
+Abra su navegador y visite http://localhost:1313 para acceder a su copia local del sitio. A medida que vaya haciendo cambios en el código fuente, Hugo irá actualizando la página y forzará la actualización en el navegador.
```
[the remark this answers is not in the document]
Comment (raelga) [reply README-es.md]: Pone que es Hugo: `Hugo irá actualizando..`

File: README-es.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Bienvenido!
+Este repositorio contiene todos los recursos necesarios para construir el [sitio web de Kubernetes y su documentación](https://kubernetes.io/). ¡Estamos encantados de que quiera contribuir!
+
+## Contribuyendo con la documentación
+
+Puede hacer clic en el botón **Fork** situado en la esquina superior derecha de la pantalla para crear una copia de este repositorio en su cuenta de GitHub. A ese tipo de copia se le llama **Fork** (bifurcación) y le permite editar el contenido del repositorio en su copia personal. Realice los cambios que quiera en su **fork** y, cuando esté listo para enviarnos esos cambios, vaya a **fork** en repositorio en su GitHub y cree una nueva **pull request** para que tengamos constancia de la propuesta de cambios.
```
[the remark this answers is not in the document]
Comment (raelga) [reply README-es.md]: En este caso,  esté se acentúa https://dle.rae.es/?id=f3Kx0yH.

File: README-es.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+Una vez la **pull request** ha sido creada, un revisor de Kubernetes asumirá la responsabilidad de proporcionar comentarios claros y prácticos sobre como continuar. Como propietario de la PR, **es su responsabilidad modificar la pull request para abordar los comentarios proporcionados por el revisor de Kubernetes.** Tenga en cuenta que algunas veces puede terminar teniendo más de un revisor de Kubernetes participando en la revisión, por lo que es posible que reciba comentarios de más revisores a parte del asignado originalmente en la creación de la **pull request**. Además, en algunas ocasiones, es posible que uno de sus revisores solicite una revisión técnica por parte de un [revisor técnico de Kubernetes](https://github.com/kubernetes/website/wiki/Tech-reviewers) si lo considera necesario.
```
[the remark this answers is not in the document]
Comment (raelga) [reply README-es.md]: Yo juraría que es femenino, de `La “petición de estirar cambios”?` En el resto del parágrafo se usa el femenino `una nueva`, `modificar la`, ...

File: content/es/community/static/cncf-code-of-conduct.md (hunk 7 above)
Comment (raelga) [reply content/es/community/static/cncf-code-of-conduct.md]: Update requested in https://github.com/cncf/foundation/pull/35

File: content/es/_index.html (hunk 4 above)
Comment (raelga) [reply content/es/_index.html]: Fixed in https://github.com/kubernetes/website/pull/13543/commits/bd9afbbbfecd2e24628c525f1d166134bd18dc0e

File: content/es/community/_index.html (hunk 5 above)
Comment (raelga) [reply content/es/community/_index.html]: Fixed in https://github.com/kubernetes/website/pull/13543/commits/e0ef1a231dd9935e8647feff4c4b04f0b32e4010

File: content/es/community/_index.html (hunk 6 above)
Comment (raelga) [reply content/es/community/_index.html]: Fixed in https://github.com/kubernetes/website/pull/13543/commits/b94d411982f46438193abbb356901cf4a77c3fdd

File: content/es/docs/sitemap.md (hunk 8 above)
Comment (raelga) [reply content/es/docs/sitemap.md]: Fixed in https://github.com/kubernetes/website/pull/13543/commits/ec0c232821f565d0f719caed01d78f02e6400421

Comment (raelga) [reply content/es/docs/sitemap.md]: Fixed in https://github.com/kubernetes/website/commit/21c69932e27b921cb657a09d7e7ff17e2a8938f0

File: content/es/community/static/cncf-code-of-conduct.md (hunk 7 above)
Comment (raelga) [reply content/es/community/static/cncf-code-of-conduct.md]: Fixed in https://github.com/kubernetes/website/commit/21c69932e27b921cb657a09d7e7ff17e2a8938f0

**Comment by sieben** [comment]
@zacharysarah Could you merge it?

**Comment by sieben** [comment]
Is a squash required?

**Comment by zacharysarah** [comment]
@sieben Yes, squashing is preferred in general. In this case, I see no reverts or other problematic commits, so I think we can go ahead and merge it.

**Comment by zacharysarah** [comment]
@raelga 👋 This looks great! I will go ahead and merge this now. Some things to follow up with:

1. Open a PR in k/org to add [your new aliases](https://github.com/kubernetes/website/pull/13543/files#diff-0de52d14f35759dfa67139f90c82bd7fR51) to [the list of SIG Docs teams in k/org](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L14). This will automate the creation and maintenance of GitHub teams that folks can `@mention` in issues and PRs.

2. In the same file/PR, add the individual members of `sig-docs-es-owners` to the [`sig-docs-l10n-admins`](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L122) and [`website-milestone-maintainers`](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L195) sections.

Please `/assign` me to the PR you create in k/org. Let me know if you have questions about these steps, I'm happy to help. 

In the meantime, congratulations! 🎉 

/lgtm
/approve

**Comment by raelga** [comment]
Thanks a lot for all your support and help @zacharysarah!

The k/org PR is https://github.com/kubernetes/org/pull/685

**Comment by seomago** [comment]
CLA signed

**Comment by raelga** [comment]
Tracked in https://github.com/kubernetes/website/issues/13948

**Pull request merged**